src/devtools/client/debugger/src/actions/breakpoints/breakpoints.ts

```diff
@@ -39,13 +39,7 @@ export function removeBreakpointsAtLine(
 
 export function removeBreakpoint(cx: Context, breakpoint: Breakpoint): UIThunkAction {
   return async dispatch => {
-    if (isLogpoint(breakpoint)) {
```

Comment:
This fixes it because `removeBreakpointOption` does not actually remove the print statement it just removes the options...